packages/Algernon.package/ALGWorldMenuItem.class/instance/icon.st

```diff
@@ -2,5 +2,5 @@ icon
 icon
 
 	^ self class mapping 
```

Comment:
I think we should come up with a better name than "mapping", at least something like `iconMapping` or so